mne/io/base.py

```diff
@@ -1542,12 +1546,12 @@ def resample(self, sfreq, npad='auto', window='boxcar', stim_picks=None,
 
             return self
         else:
-            if copy:
```

Comment:
This is effectively testing whether the `copy`-module has been imported?! Perhaps a copy-parameter was intended?